src/site/articles/mixins/index.markdown

```diff
@@ -17,13 +17,11 @@ article:
 # {{ page.title }}
 
 _Written by Gilad Bracha <br>
-December 2012 (updated November 2013)_
+December 2012 (updated October 2015)_
 
-This document describes mixins in Dart.
-The semantics are deliberately restricted in several ways,
-so as to reduce disruption to our existing implementations,
-while allowing future evolution toward a full-fledged mixin implementation.
-This restricted version already provides considerable value.
+This document describes mixins in Dart. We recently relaxed
+some of the restrictions of early implementations; this document
+describes the current state of play.
```

Comment:
Could you define this a little better? What releases support the relaxed restrictions? Anything else the reader ought to know about using them?